pandas/tests/indexes/multi/test_format.py

```diff
@@ -103,3 +83,126 @@ def test_repr_max_seq_item_setting(idx):
     with pd.option_context("display.max_seq_items", None):
         repr(idx)
         assert '...' not in str(idx)
+
+
+class TestRepr(object):
```

Comment:
We just removed all inheritance from object in another PR; should do so here as well

Reply:
Yeah, I`ll  remove this when I rebase the next time.